### PR TITLE
Text updates (wip)

### PR DESCRIPTION
**Pull request by JasonP01**
Changes a vast amount of the server to ingame texts. Such as:

- History command syntax
- Votekick announcement
- QoL by removing ‛ from embeds

### CODE REVIEW

**Comment by JasonP01** [comment]
Ill resolve the conflicts eventually, very annoying >:(

File: imperium-mindustry/src/main/kotlin/com/xpdustry/imperium/mindustry/security/VoteKickCommand.kt [hunk 1]
```diff
@@ -163,6 +163,7 @@ class VoteKickCommand(instances: InstanceManager) :
 
     override suspend fun onVoteSessionFailure(session: VoteManager.Session<Context>) {
         runMindustryThread { freezes.setTemporaryFreeze(session.objective.target, null) }
+        session.objective.target.sendMessage("[sky]You are involved in a vote kick so you will be temporarily frozen. Being frozen prevents you from building or interacting. You will be un-frozen once the vote kick ends."
```
Comment (JasonP01) [inline imperium-mindustry/src/main/kotlin/com/xpdustry/imperium/mindustry/security/VoteKickCommand.kt:166]: @phinner this works right?

**Review by JasonP01 — COMMENTED** [review]
@phinner this works right?

File: imperium-mindustry/src/main/kotlin/com/xpdustry/imperium/mindustry/security/VoteKickCommand.kt (hunk 1 above)
Comment (phinner) [reply imperium-mindustry/src/main/kotlin/com/xpdustry/imperium/mindustry/security/VoteKickCommand.kt:166]: You are telling the target they are still frozen while :
- the line above disables the temporary freeze.
- `onVoteSessionFailure` runs only after a failed vote, so why still freezing ?

Comment (JasonP01) [reply imperium-mindustry/src/main/kotlin/com/xpdustry/imperium/mindustry/security/VoteKickCommand.kt:166]: Damn i looked at the wrong line i guess :( ill fix it!

**Comment by JasonP01** [comment]
@phinner could u possible tell me what exactly the conflict is, as to me there is 0 reason for the conflict to exist 

To resolve it i literally copy pasted your file, then changed the text back (~~preferably maybe fix it to, its so annoying~~)

**Comment by JasonP01** [comment]
Closing cause im not going to be fighting conflicts, ill remake this on a fresh branch some other day

**Pull request closed without merging**